README.md

```diff
@@ -1,44 +1,78 @@
-# What The Git
 
-## Introduction
+![What The Git Logo](public/logo.webp)
 
-What The Git is a web application that makes it easy to find what a git command does by just typing it into the search bar.
 
-It is written with Typescript using React, works only with a front-end, using parsing of command-line arguments and a simple .json file containing all the descriptions of the commands and their flags. It is also easy to self-host since it doesn't contain a server.
 
-## Features
+<h4 align="center">A web application that makes it easy to find what a git command does by just typing it into the search bar. </h4>
+
+<!-- <p align="center">
+  <a href="https://badge.fury.io/js/electron-markdownify">
+    <img src="https://badge.fury.io/js/electron-markdownify.svg"
+         alt="Gitter">
+  </a>
+  <a href="https://gitter.im/amitmerchant1990/electron-markdownify"><img src="https://badges.gitter.im/amitmerchant1990/electron-markdownify.svg"></a>
+  <a href="https://saythanks.io/to/amitmerchant1990">
+      <img src="https://img.shields.io/badge/SayThanks.io-%E2%98%BC-1EAEDB.svg">
+  </a>
+  <a href="https://www.paypal.me/AmitMerchant">
+    <img src="https://img.shields.io/badge/$-donate-ff69b4.svg?maxAge=2592000&amp;style=flat">
+  </a>
+</p> -->
+
+<p align="center">
+  <a href="#features">Features</a> •
+  <a href="#upcoming-features">Upcoming Features</a> •
+  <a href="#supported-commands">Supported Commands</a> •
+  <a href="#contribute">Contribute</a>  
+</p>
```

Comment:
I like this a lot too! Could you maybe make the table of contents to be one item on each line? I think it would be easier to click if all the items are on a separate line.